extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -101,8 +101,9 @@ export class RefreshManager {
   /**
    * @param {!./amp-a4a.AmpA4A} a4a The AmpA4A instance to be refreshed.
    * @param {!RefreshConfig} config
+   * @param {number=} refreshIntervalSecs Refresh interval in seconds.
    */
-  constructor(a4a, config) {
+  constructor(a4a, config, refreshIntervalSecs) {
```

Comment:
Mixing between seconds and milliseconds seems dangerous to me... too easy to pass the wrong value.  Why not just force all to millis (including the header)?

Reply:
Done.